src/Spd.Resource.Repository/Licence/LicenceRepository.cs

```diff
@@ -18,11 +18,14 @@ internal class LicenceRepository : ILicenceRepository
     public async Task<LicenceResp> GetAsync(Guid licenceId, CancellationToken ct)

```

Comment:
Do we need to modify the `LicenceResp` object considering that we are pulling more data?

Reply:
no, resp already contains more info. If we do not pulling more data, we cannot fill up resp.